sql/mob_droplist.sql

```diff
@@ -6643,8 +6643,8 @@ INSERT INTO `mob_droplist` VALUES (1283,0,0,1000,4788,110);
 INSERT INTO `mob_droplist` VALUES (1283,0,0,1000,4814,60);
 INSERT INTO `mob_droplist` VALUES (1284,0,0,1000,19208,100);
 INSERT INTO `mob_droplist` VALUES (1285,2,0,1000,656,0);
-INSERT INTO `mob_droplist` VALUES (1285,0,0,1000,16755,900);
-INSERT INTO `mob_droplist` VALUES (1285,0,0,1000,17184,100);
+INSERT INTO `mob_droplist` VALUES (1285,1,1,1000,16755,900);
+INSERT INTO `mob_droplist` VALUES (1285,1,1,1000,17184,100);
```

Comment:
I'd argue that archer's knife drop should be closer to 8% rather than 10% according to ffxidb.  Other one at 92% is safe.